src/nodejs/unit-http/http.js

```diff
@@ -5,13 +5,21 @@
 
 'use strict';
 
+const { stderr } = require('node:process');
 const {
     Server,
     ServerRequest,
     ServerResponse,
 } = require('./http_server');
 
-function createServer (requestHandler) {
+function createServer (options, requestHandler) {
+    if (typeof options === 'function') {
+        requestHandler = options;
+        options = {};
+    } else {
+        stderr.write('createServer was called with an options object. Unit ignores this for the time being.');
```

Comment:
I wonder if we should move this error handling into the `Server` constructor, and always pass `options` to it, even if empty? That'd potentially catch more attempts to customize the server.

Reply:
Here's the Server constructor in Node: https://github.com/nodejs/node/blob/v21.6.1/lib/_http_server.js#L520

Reply:
Option handling moved into the Server constructor in [52550b1](https://github.com/nginx/unit/pull/1091/commits/52550b18897802b5820bec809902513195afe4af)